lib/coinatmradar/coinatmradar.js

```diff
@@ -28,7 +28,7 @@ function mapCoin (info, deviceId, cryptoCode) {
   const cashInFee = showRates ? cryptoConfig.cashInCommission / 100 : null
   const cashOutFee = showRates ? cryptoConfig.cashOutCommission / 100 : null
   const cashInRate = showRates ? rates.cashIn.toNumber() : null
-  const cashOutRate = showRates ? rates.cashOut.toNumber() : null
+  const cashOutRate = showRates && rates.cashOut? rates.cashOut.toNumber() : null
```

Comment:
Try: 

```javascript
const cashOutRate = showRates ? _.invoke('cashOut.toNumber', rates) : null
```

Also, might as well fix this for cashInRate, as well.

Reply:
lodash elegance.
Thank you.